### PR TITLE
Set the prefix of the URL field to https: //

### PR DESCRIPTION
**Pull request by datnguyenhuy**
https://user-images.githubusercontent.com/56970970/117961884-d192ad00-b348-11eb-97c7-276ed6a9f158.mp4

### CODE REVIEW

File: lib/presentation/localizations/app_localizations.dart [hunk 1]
```diff
@@ -70,6 +70,11 @@ class AppLocalizations {
       name: 'https');
   }
 
+  String get url {
+    return Intl.message('URL',
+        name: 'url');
+  }
```
Comment (imGok) [inline lib/presentation/localizations/app_localizations.dart:76]: Can you indent like previous functions please :(

Comment (datnguyenhuy) [reply lib/presentation/localizations/app_localizations.dart:76]: I fixed and pushed a new commit. Thank you, please

Comment (hoangdat) [reply lib/presentation/localizations/app_localizations.dart:86]: don't need the `URL`. Use the same `https:// `

File: lib/presentation/widget/login/login_widget.dart [hunk 2]
```diff
@@ -230,15 +232,20 @@ class _LoginWidgetState extends State<LoginWidget> {
         (failure) {
           if (_checkUrlError(failure)) {
               return LoginInputDecorationBuilder()
-                  .hintText(AppLocalizations.of(context).https)
+                  .labelText(AppLocalizations.of(context).url)
```
Comment (hoangdat) [inline lib/presentation/widget/login/login_widget.dart:235]: use `AppLocalizations.of(context).https`

File: lib/presentation/widget/login/login_widget.dart [hunk 3]
```diff
@@ -230,15 +232,20 @@ class _LoginWidgetState extends State<LoginWidget> {
...
+                  .prefixText(AppLocalizations.of(context).https)
                   .enabledBorder(OutlineInputBorder(
                       borderRadius: BorderRadius.all(Radius.circular(6)),
                       borderSide: BorderSide(
                       width: 1, color: AppColor.loginTextFieldErrorBorder)));
           }
-          return LoginInputDecorationBuilder().hintText(AppLocalizations.of(context).https);
+          return LoginInputDecorationBuilder()
+              .labelText(AppLocalizations.of(context).url)
```
Comment (hoangdat) [inline lib/presentation/widget/login/login_widget.dart:243]: same

File: lib/presentation/widget/login/login_widget.dart [hunk 4]
```diff
@@ -230,15 +232,20 @@ class _LoginWidgetState extends State<LoginWidget> {
...
+              .prefixText(AppLocalizations.of(context).https);
         },
-        (success) => LoginInputDecorationBuilder().hintText(AppLocalizations.of(context).https));
+        (success) => LoginInputDecorationBuilder()
+            .labelText(AppLocalizations.of(context).url)
```
Comment (hoangdat) [inline lib/presentation/widget/login/login_widget.dart:247]: same

File: lib/presentation/widget/login/login_widget.dart [hunk 5]
```diff
@@ -170,7 +171,8 @@ class _LoginWidgetState extends State<LoginWidget> {
                                       .textDecoration(
                                       _buildCredentialInputDecoration(
                                           viewState,
-                                          AppLocalizations.of(context).password))
+                                          AppLocalizations.of(context).password,
+                                          AppLocalizations.of(context).enter_your_password))
```
Comment (hoangdat) [inline lib/presentation/widget/login/login_widget.dart:175]: use `AppLocalizations.of(context).password`

**Review by dab246 — APPROVED** [review]
LGTM

**Review by huynguyennovem — APPROVED** [review]
look good!

**Pull request merged**